.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,69 @@
+name: Build Wheels
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+       - '*'
+  pull_request:
+    branches:
+      - master
+
+  workflow_dispatch:
+
+env:
+  # Only support 64-bit CPython >= 3.6
```

Comment:
I suggest to only support `>= 3.7`

Python 3.6 is not maintained anymore since December 2021. See https://devguide.python.org/devcycle/#end-of-life-branches